src/components/MultiStoryContainer/hooks/useMultiStoryContainer.ts

```diff
@@ -59,13 +66,44 @@ const useMultiStoryContainer = (
     !isScrollActiveRef.current && updateStoryIndex();
   };
 
+  const layoutValue = useSharedValue(0);
+
+  useEffect(() => {
+    if (visible) {
+      layoutValue.value = withTiming(1, { duration: 400 }); // Note: 400 is the duration of the animation while open the modal
```

Comment:
@prince-d-simform We should revise this, and remove Note:
e.g Default open Modal animation duration is 400